activemodel/test/cases/secure_password_test.rb

```diff
@@ -212,6 +212,11 @@ class SecurePasswordTest < ActiveModel::TestCase
     assert_equal @user, @user.authenticate_recovery_password("42password")
   end
 
+  test "authenticating user with a blank password digest" do
```

Comment:
I think the description could maybe better explain the desired behaviour.
What do you think about mentioning it should return false?
```suggestion
  test "authenticate returns false when password digest is blank" do
```

Or maybe even mention it should not raise as well?
```suggestion
  test "authenticate should return false and not raise when password digest is blank" do
```

Reply:
Sorry, I'm not a native English speaker. But I like your suggestions, I will try to follow the same pattern in the future. :heart: